omod/src/main/java/org/openmrs/web/controller/ConceptFormController.java

```diff
@@ -678,6 +697,161 @@ public Concept getConceptFromFormData() {
 			return concept;
 		}
 		
+		/**
+		 * This method sets reference ranges to concept numeric.
+		 * 
+		 * @param cn ConceptNumeric
+		 * @since 1.17.0
+		 */
+		private void setConceptReferenceRanges(ConceptNumeric cn) {
+			if (this.referenceRanges == null) {
+				return;
+			}
+			for (ConceptReferenceRange referenceRange : this.referenceRanges) {
+				if (referenceRange == null) {
+					continue;
+				}
+				
+				if (referenceRange.getId() != null) {
+					if (referenceRange.getId() <= 0) {
+						removeReferenceRange(cn, referenceRange);
+					} else {
+						updateReferenceRange(cn, referenceRange);
+					}
+				} else {
+					if (referenceRange.getHiAbsolute() != null && referenceRange.getLowAbsolute() != null) {
+						addReferenceRange(cn, referenceRange);
+					}
+				}
+			}
+		}
+		
+		/**
+		 * This method removes a reference range from conceptNumeric
+		 * 
+		 * @param cn conceptNumeric
+		 * @param referenceRange referenceRange
+		 * @since 1.17.0
+		 */
+		private void removeReferenceRange(ConceptNumeric cn, ConceptReferenceRange referenceRange) {
+			try {
+				Object platformReferenceRange = new ConceptFormMapper().mapToConceptReferenceRange(referenceRange, cn);
+				setMethodValue(cn, "removeReferenceRange", platformReferenceRange);
+			}
+			catch (Exception exception) {
+				logger.warn("Failed to remove concept reference range. "
```

Comment:
The logs will be filled with many of these warnings for those running platform versions lower than 2.7
I do not find it a good practice to fill logs with such warning where there is completely nothing wrong about running lower versions of the platform.